documentation/book/proc-deploying-kafka-cluster-kubernetes.adoc

```diff
@@ -5,26 +5,28 @@
 [id='deploying-kafka-cluster-kubernetes-{context}']
 = Deploying the Kafka cluster to {KubernetesName}
 
+The following procedure describes how to deploy an ephemeral or persistent Kafka cluster to {KubernetesName} on the command line.
+
 .Prerequisites
 
-* Before deploying a Kafka cluster, the Cluster Operator must be deployed.
+* You must have deployed the Cluster Operator before you can deploy a cluster.
 
 .Procedure
 
-. If you are planning to use the Kafka broker for development or testing, create an ephemeral cluster
+. If you plan to use the cluster for development or testing purposes, create and deploy an ephemeral cluster using `kubectl apply`.
```

Comment:
I know you are just refactoring an existing sentence. But I wonder is this is the right think to say. People can use also persistent clusters for development and testing. There is no requirement to do that. So I think we need to change this sentence to be less _commanding_. I don't know, maybe:

_If you plan to use the cluster for development or testing purposes, you can create and deploy an ephemeral cluster using `kubectl apply`._

Reply:
Yeah, I see what you mean here. The imperative mood can be used when an action _must_ be performed. As you say, it's a bit too commanding in this case!